### PR TITLE
Add some helper functions to get LVM VDO mode and state strings

### PR DESCRIPTION
**Pull request by vojtechtrefny**

### CODE REVIEW

File: src/plugins/lvm-dbus.c [hunk 1]
```diff
@@ -3567,3 +3567,83 @@ gboolean bd_lvm_vdo_pool_resize (const gchar *vg_name, const gchar *pool_name, g
...
+const gchar* bd_lvm_get_vdo_operating_mode_str (BDLVMVDOOperatingMode mode, GError **error) {
+    switch (mode) {
+    case BD_LVM_VDO_MODE_RECOVERING:
+        return "recovering";
+    case BD_LVM_VDO_MODE_READ_ONLY:
+        return "read-only";
+    case BD_LVM_VDO_MODE_NORMAL:
+        return "normal";
+    case BD_LVM_VDO_MODE_UNKNOWN:
+        return "unknow";
```
Comment (tbzatek) [inline src/plugins/lvm-dbus.c]: Missing 'n'. One more occurrence below.d

Comment (vojtechtrefny) [reply src/plugins/lvm-dbus.c]: Fixed.

**Comment by vojtechtrefny** [comment]
> Do we really care about returning an error? Wouldn't simple NULL be enough to indicate an invalid value?

It's not necessary, but it doesn't hurt. And setting the error will result in an exception in python which is better than returning `None`, IMHO.

**Comment by tbzatek** [comment]
> And setting the error will result in an exception in python which is better than returning `None`, IMHO.

I see, good point.

**Comment by vojtechtrefny** [comment]
Jenkins, test this please.

**Comment by vojtechtrefny** [comment]
Jenkins, test this please.

**Comment by vojtechtrefny** [comment]
Jenkins, test this please.

**Pull request merged**